Apps/Src/RTOSPedals.c

```diff
@@ -0,0 +1,40 @@
+/* Copyright (c) 2020 UT Longhorn Racing Solar */
+
+#include "RTOSPedals.h"
+
+uint8_t PreviousAccel;
+uint8_t PreviousBrake;
```

Comment:
These don't need to be global. They can be locally defined like `err` before the while loop